extensions/spark/README.md

```diff
@@ -23,3 +37,38 @@ You can pull the module from the central Maven repository by including the follo
     <version>0.20.0</version>
 </dependency>
 ```
+
+## Usage
+
+Using the DJL Spark Extension is simple and straightforward. Here is an example of how to use it to run image classification on a large dataset using Apache Spark and DJL:
+
+### Scala
+
+```scala
+import ai.djl.spark.SparkTransformer
+import ai.djl.spark.translator.SparkImageClassificationTranslator
+
+val transformer = new SparkTransformer[Classifications]()
+  .setInputCol("input_col")
+  .setOutputCol("value")
+  .setModelUrl("model_url")
+  .setOutputClass(classOf[Classifications])
+  .setTranslator(new SparkImageClassificationTranslator())
+val outputDf = transformer.transform(df)
+```
+
+### Python
+
+```python
+from djl_spark.transformer import SparkTransformer
+from djl_spark.translator import SparkImageClassificationTranslator
+
+transformer = SparkTransformer(input_col="input_col",
+                               output_col="value",
+                               model_url="model_url",
+                               output_class="ai.djl.modality.Classifications",
+                               translator=SparkImageClassificationTranslator())
```

Comment:
We may also need a Python generic Python Translator. User can implements based on that. Inside the python translator, we embedded the numpy translator in Java. Where we accept a numpy array input and return as numpy array outputs.